Rakefile

```diff
@@ -277,9 +278,10 @@ REGEN_METADATA = {
         tag: 'arm_mobile'
     },
     azure_mgmt_network: {
-        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-network/2016-09-01/swagger/network.json',
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-network/compositeNetworkClient.json',
```

Comment:
Please also update [swagger_to_sdk_config.json](https://github.com/Azure/azure-sdk-for-ruby/blob/master/swagger_to_sdk_config.json#L276).